pkg/standalone/run.go

```diff
@@ -47,39 +47,46 @@ const (
 // RunConfig represents the application configuration parameters.
 type RunConfig struct {
```

Comment:
in a follow up PR, will move RunConfig out of Standalone package into its own package.